django/publicmapping/run_django.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Run celery worker in background
+# TODO: Don't use C_FORCE_ROOT
+C_FORCE_ROOT=1 python manage.py celery worker --loglevel=info &
```

Comment:
Running multiple processes in a single Docker container is generally [considered a no-no](https://docs.docker.com/engine/admin/multi-service_container/). In this case we could set up a `celery` service that uses the `image` parameter to ensure that it works off the same image as the `django` service.

Reply:
Yeah, it's bad, but so many things about this PR are bad.

I think that to have a standalone celery service we'd need to upgrade celery and its API has breaking changes. That would also necessitate doing away with `django-celery` which doesn't support new celery.

I'm not sure what to do with this PR at this point. It almost feels like a spike in that now we know everything that's involved in getting it working.

Do you think the best course of action would be to let it serve as a reference and just open up a new PR with "reporting done right"?